resilience4j-spring6/src/main/java/io/github/resilience4j/spring6/bulkhead/configure/BulkheadAspect.java

```diff
@@ -116,8 +116,8 @@ public Object bulkheadAroundAdvice(ProceedingJoinPoint proceedingJoinPoint,
                 () -> proceedInThreadPoolBulkhead(proceedingJoinPoint, methodName, returnType, backend);
             return fallbackExecutor.execute(proceedingJoinPoint, method, bulkheadAnnotation.fallbackMethod(), bulkheadExecution);
         } else {
-            io.github.resilience4j.bulkhead.Bulkhead bulkhead = getOrCreateBulkhead(methodName,
-                backend);
+            String configKey = bulkheadAnnotation.configuration().isEmpty() ? backend : bulkheadAnnotation.configuration();
```

Comment:
I think when `bulkheadAnnotation.type() == Bulkhead.Type.THREADPOOL` the `backend` is not overridden when the config key is set. I think this is missing.

Reply:
In fact threadpool bulkhead was always using the default config.
I've completed implementation.
Many thx for your review.